lib/matplotlib/axis.py

```diff
@@ -1477,9 +1479,14 @@ def have_units(self):
         return self.converter is not None or self.units is not None
 
     def convert_units(self, x):
```

Comment:
This seems to be converting anything numlike, which includes things like `pint.Quantity`, into an array/masked array before calling `convert`. That seems like it would break `pint`, though I will note that our unit tests seem to be passing.

Reply:
Great, I’ll check vs pint when I look at the other comments above.  Thanks for the reminder...

Reply:
Ok, I guess I still don't understand how all of this works. 😉 So `is_numlike` only returns `true` for things that are basic numbers (or contain them) and don't need converting. I think I'm ok with this code then. Tests passing is always a good thing too. 😄